tests/pester-tests/commands/choco-source.Tests.ps1

```diff
@@ -67,6 +67,45 @@ Describe "choco <_>" -ForEach $Command -Tag Chocolatey, SourceCommand {
         }
     }
 
+    Context "Add source that already exists" {
+        BeforeAll {
+            $null = Disable-ChocolateyFeature -Name "useEnhancedExitCodes"
+
+            # Ensure source is not available
+            Invoke-Choco $CurrentCommand add --name "already-exists" --source "https://somewhere/out/there/"

```

Comment:
The comment seems counter to what the action actually is. Although it's not necessary to assign `Invoke-Choco` to `$null`, it's become the norm in the tests, so let's keep it going here.

```suggestion
            # Ensure source is already added
            $null = Invoke-Choco $CurrentCommand add --name "already-exists" --source "https://somewhere/out/there/"

```

Reply:
As discussed, I have fixed this up.